SPECS/cert-manager/cert-manager.spec

```diff
@@ -1,96 +1,117 @@
-#   1. wget https://github.com/jetstack/%{name}/archive/refs/tags/v%{version}.tar.gz -o %%{name}-%%{version}.tar.gz
-#   2. tar -xf %%{name}-%%{version}.tar.gz
-#   3. cd %%{name}-%%{version}
-#   4. patch -p1 < Fix-dependency-checksum.patch
-#   5. mkdir -p BAZEL_CACHE
-#   6. bazel fetch --repository_cache=BAZEL_CACHE //...
-#   7. tar  --sort=name \
-#           --mtime="2021-04-26 00:00Z" \
-#           --owner=0 --group=0 --numeric-owner \
-#           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
-#           -cf %%{name}-%%{version}-vendor.tar.gz BAZEL_CACHE
-Source1:        %{name}-%{version}-vendor.tar.gz
-# Below is a manually created tarball, no download link.
 # We're using pre-populated GO dependencies from this tarball, since network is disabled during build time.
 #   1. wget https://github.com/jetstack/%{name}/archive/refs/tags/v%{version}.tar.gz -o %%{name}-%%{version}.tar.gz
 #   2. tar -xf %%{name}-%%{version}.tar.gz
 #   3. cd %%{name}-%%{version}
-#   4. go mod download
-#   5. cd $HOME
+#   4. go mod vendor
 #   5. tar  --sort=name \
-#           --mtime="2021-04-26 00:00Z" \
+#           --mtime="2022-08-10 00:00Z" \
 #           --owner=0 --group=0 --numeric-owner \
 #           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
-#           -cf %%{name}-%%{version}-gocache.tar.gz go
-Source2:        %{name}-%{version}-gocache.tar.gz
-Patch0:         Fix-dependency-checksum.patch
-Patch1:         Fix-os-WriteFile.patch
-BuildRequires:  bazel
-BuildRequires:  git
-BuildRequires:  golang <= 1.17.10
+#           -cf %%{name}-%%{version}-govendor.tar.gz vendor
+Source1:        %{name}-%{version}-govendor.tar.gz
+BuildRequires:  golang
 BuildRequires:  patch
 
 %description
 cert-manager is a Kubernetes add-on to automate the management and issuance
 of TLS certificates from various issuing sources.
 
+%package acmesolver
+Summary: cert-manager's acmesolver binary
+
+%description acmesolver
+HTTP server used to solve ACME challenges.
+
+%package cainjector
+Summary: cert-manager's cainjector binary
+
+%description cainjector
+cert-manager CA injector is a Kubernetes addon to automate the injection of CA data into
+webhooks and APIServices from cert-manager certificates.
+
+%package controller
+Summary: cert-manager's controller binary
+
+%description controller
+cert-manager is a Kubernetes addon to automate the management and issuance of
+TLS certificates from various issuing sources.
+
+%package cmctl
+Summary: cert-manager's cmctl binary
+
+%description cmctl
+cmctl is a CLI tool manage and configure cert-manager resources for Kubernetes
+
+%package webhook
+Summary: cert-manager's webhook binary
+
+%description webhook
+Webhook component providing API validation, mutation and conversion functionality for cert-manager.
+
 %prep
 %autosetup -p1
 %setup -q -T -D -a 1
 
-%build
```

Comment:
Let's keep building inside the `%build` section instead of moving things to `%prep`. We don't really expect too much to happen there.

Reply:
Good catch.

Reply:
Fixed.